sdks/java/core/src/main/java/org/apache/beam/sdk/runners/PTransformMatcher.java

```diff
@@ -20,12 +20,16 @@
 
 import org.apache.beam.sdk.annotations.Experimental;
 import org.apache.beam.sdk.annotations.Experimental.Kind;
+import org.apache.beam.sdk.annotations.Internal;
 import org.apache.beam.sdk.transforms.AppliedPTransform;
 import org.apache.beam.sdk.transforms.PTransform;
 
 /**
- * Matches applications of {@link PTransform PTransforms}.
+ * <b><i>For internal use only; no backwards-compatibility guarantees.</i></b>
```

Comment:
I almost did, but `PipelineRunner` is public. It should really just be toplevel.